src/apis/dockerhub/Dockerhub.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime, timedelta, UTC
+import json
+import logging
+
+import requests
+from pydantic import Field
+
+from src.apis.general.Api import ApiFetcher
+from src.apis.general.PaginationSettings import PaginationSettings
+from src.apis.general.StopPaginationSettings import StopPaginationSettings
+
+logger = logging.getLogger(__name__)
+
+
+class DockerHub(ApiFetcher):
+    """
+    :param dockerhub_token: The DockerHub personal access token or password
+    :param pagination_off: True if pagination should be off, False otherwise
+    :param days_back_fetch: Amount of days to fetch back in the first request, Optional (adds a filter on 'from')
+    :param page_size: Number of events to return in a single request (for pagination)
```

Comment:
this is not implemented in the request, maybe we can remove the doc or add a reference to it when building the request